<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fightpage</class>
 <widget class="QWidget" name="Fightpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QFrame" name="blueside">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="blue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tondu</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Master:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueNickname">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Pokemon Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bluePokemonName">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueLevel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Race:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueRace">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Rarity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueRarity">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Attack:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueBaseAttack">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Defense:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueDefense">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Wsp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blueWsp">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="blueHp">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="redside">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>961</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="redNickname">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Pokemon Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="redPokemonName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="redLevel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Race:</string>
    </property>
   </widget>
   <widget class="QLabel" name="redRace">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Rarity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="redRarity">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Attack:</string>
    </property>
   </widget>
   <widget class="QLabel" name="redBaseAttack">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Defense:</string>
    </property>
   </widget>
   <widget class="QLabel" name="redDefense">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Wsp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="redWsp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
     </font>
    </property>
    <property name="text">
     <string>Master:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>16</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="blue_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tondu</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="redHp">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="bluefightinfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="redfightinfo">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
